<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stochastic Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="text"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLineEdit" name="plainTextEdit">
          <property name="toolTip">
           <string>نماد مورد نظر را به همراه جفت ارز و با حروف بزرگ وارد نمایید.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>نماد به همراه جفت ارز (حروف بزرگ):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>1m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1d</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1w</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>تایم فریم:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="plainTextEdit_3">
          <property name="toolTip">
           <string>نماد مورد نظر را به همراه جفت ارز و با حروف بزرگ وارد نمایید.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>دورهٔ K استوکاستک:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="plainTextEdit_4">
          <property name="toolTip">
           <string>نماد مورد نظر را به همراه جفت ارز و با حروف بزرگ وارد نمایید.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>مقدار هموارسازی: </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="plainTextEdit_5">
          <property name="toolTip">
           <string>نماد مورد نظر را به همراه جفت ارز و با حروف بزرگ وارد نمایید.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>دورهٔ D استوکاستیک: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>نمایش چارت</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeveloped_By_TheDevileytion">
    <property name="title">
     <string>Developed By TheDevileytion</string>
    </property>
   </widget>
   <addaction name="menuDeveloped_By_TheDevileytion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
